src/main/webapp/i18n/de/assessment.json

```diff
@@ -17,6 +17,14 @@
             "moreFeedbackRequest": "Du musst auf die Feedback Anfrage antworten.",
             "assessmentReadOnlyHandledComplaint": "Die Beschwerde über deine Bewertung wurde bearbeitet.",
             "assessmentDueDateIsOver": "Die Einreichungsfrist der Bewertung ist vorbei, die nächste Einreichung wird direkt veröffentlicht",
+            "diffView": {
+                "differenceActivate": "Aktiviere Diff-Ansicht",
```

Comment:
does not sound good. Maybe use "Aktiviere Vergleichsansicht"?

Reply:
yes, sounds good